beetsplug/fetchart.py

```diff
@@ -219,6 +221,17 @@ def fetch_image(self, candidate, plugin):
         pass
 
 
+class EmbeddedArtSource(ArtSource):
```

Comment:
A docstring here might be nice.

Reply:
I added docstrings to the methods defined in this class.  The other ArtSource classes don't have class docstrings so I didn't add one to Embedded either.  Let me know if you'd like more.